docs/notebooks/scverse_data_interoperability.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>As noted before, the javascript headline is not rendered correctly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>